glide-core/tests/test_client.rs

```diff
@@ -316,6 +316,45 @@ pub(crate) mod shared_client_tests {
     #[rstest]
     #[timeout(SHORT_CLUSTER_TEST_TIMEOUT)]
     fn test_request_timeout(#[values(false, true)] use_cluster: bool) {
+        block_on_all(async {
+            let mut test_basics = setup_test_basics(
+                use_cluster,
+                TestConfiguration {
+                    request_timeout: Some(1),
```

Comment:
is is in seconds or in millis? please, add a comment
```suggestion
                    request_timeout: Some(1), // millis
```